data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/resistance/resistance_affect_allies.cfg

```diff
@@ -0,0 +1,42 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [resistance]affect_allies=
+##
+# Actions:
+# Make sides 1 and 2 allies
+# Give side 1's adjacent allies 50% resistance to all damage types
+# Have side 2 attack side 4
+# Have side 3 attack side 1
+##
+# Expected end state:
+# The damage from the attack is reduced by 50% for allies
```

Comment:
... for allies, but not self

Reply:
The next line already says:
> The damage from the attack is not reduced for others

Reply:
It can be ambiguous, as "allies" is often said with the implication that it also applies to your own side, so I think it would be easier for later readers to say it explicitly.

Reply:
I'd be a bit confused by that assumption, since it doesn't make much sense to say that you're allied with yourself.

Reply:
Updated.